### ISSUE TITLE
dump is skipping /etc/gitea/app.ini

### ISSUE THREAD

**Issue by fazlerabbi37** [go-gitea/gitea#6034]
- Gitea version (or commit ref): 1.7.1
- Git version: 2.7.4
- Operating system: Ubuntu Server 16.04 LTS
- Database (use `[x]`):
  - [x] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [ ] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [x] No
  - [ ] Not relevant
- Log gist:

## Description

I installed Gitea from binary using this [Installation from binary](https://docs.gitea.io/en-us/install-from-binary/) doc. I am trying to use the ``dump`` command to take backup. But the ``dump`` command is skipping the ``app.ini`` in ``/etc/gitea`` where all of my configs are.

## Steps to reproduct
1. perform backup with following steps:

```
cd /home/git
sudo -u git /usr/local/bin/gitea dump -c /etc/gitea/app.ini
sudo mv gitea-dump-* /path/to/backup/gitea-dump.zip

```
2. now extract the zip file

```
cd /path/to/backup/
sudo unzip gitea-dump.zip

```
which gives the following output:

```
Archive:  gitea-dump.zip
  inflating: gitea-repo.zip          
  inflating: gitea-db.sql            
   creating: log/
  inflating: log/gitea.log           
  inflating: log/xorm.log            
  inflating: log/serv.log            
   creating: log/hooks/
  inflating: log/hooks/update.log    
  inflating: log/hooks/pre-receive.log  
  inflating: log/hooks/post-receive.log  
  inflating: log/http.log            
   creating: custom/
   creating: custom/conf/
  inflating: custom/conf/app.ini     

```

check for the config in ``app.ini``

```
cat custom/conf/app.ini
```
output 

```
[security]
INTERNAL_TOKEN = xxxxxxxxxxxxxxxxxxx
```

all my configurations are in ``/etc/gitea/app.ini`` not in ``custom/conf/app.ini ``. Is it expected? Does anyone know how to solve this? 

Thanks in advance!

**Comment by zeripath** [comment]
When you run the dump command do you pass in the config? E.g.

`gitea dump -c /etc/gitea/app.ini`

**Comment by fazlerabbi37** [comment]
@zeripath, yes I did.

**Comment by fazlerabbi37** [comment]
@zeripath any update? or have you been able to reproduce it?

**Comment by zeripath** [comment]
I haven't had a chance to look in to this. I guess I need some more information - what do you mean by skipping. Is it that Gitea dump isn't finding your db? Do you have a relative path as your db path? If so you must run dump from the same working directory that you normally run Gitea from.

**Comment by fazlerabbi37** [comment]
@zeripath updated the issue with Steps to reproduce.

**Comment by nicovince** [comment]
I am also having this issue with the same commands given by @fazlerabbi37 in the first message.

@zeripath do you need more informations ?

Thanks.

**Comment by mrsdizzie** [comment]
Looks like Gitea doesn't make an effort to include a configuration file specifically, so it only gets included if in the default location of custom/conf/. Will send a PR for that

**Comment by K1LLUM1N471** [comment]
Is there already a solution?

My current workaround is to add the custom configuration file /etc/gitea/app.ini with the command 'zip -r gitea-dump-...zip /etc/gitea/app.ini' in a script.

Kind regards,
K1LLUM1N471

**Comment by lafriks** [comment]
Fix will be included in v1.9.0
